logger-interface/src/logger/FileLogger.java

```diff
@@ -0,0 +1,29 @@
+package logger;
+
+import java.io.IOException;
+import java.util.logging.FileHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+
+public class FileLogger {
+
+    public void getFileLogger(String nameA, String nameB) {
+        Logger logger = Logger.getLogger(FileLogger.class.getName());
+        FileHandler fh;
+        try{
+            fh = new FileHandler("C:/temp/MyLogFile.log");
+            logger.addHandler(fh);
+            SimpleFormatter formatter = new SimpleFormatter();
+            fh.setFormatter(formatter);
+
+            logger.log(Level.SEVERE, "logging: {0} {1}", new Object[]{nameA, nameB});
+
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
we will learn about this, but catching specific exception rather than generic ones is usually better